coala_quickstart/generation/Project.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from coala_quickstart.interaction.Question import ask_question
+from coala_quickstart.Extensions import exts
+
+
+def get_project_dir(printer):
```

Comment:
maybe you could use the same studd @mr-karan used for validation? Also it doesn't make sense to rewrite the validation loop for every question.